pkg/descheduler/node/node.go

```diff
@@ -31,16 +31,23 @@ import (
 
 // ReadyNodes returns ready nodes irrespective of whether they are
 // schedulable or not.
-func ReadyNodes(client clientset.Interface, stopChannel <-chan struct{}) ([]*v1.Node, error) {
+func ReadyNodes(client clientset.Interface, nodeSelector string, stopChannel <-chan struct{}) ([]*v1.Node, error) {
 	nl := GetNodeLister(client, stopChannel)
-	nodes, err := nl.List(labels.Everything())
+
```

Comment:
Can you add a unit test to this function?

Reply:
yes will add unit tests soon.